README.md

```diff
@@ -195,5 +195,9 @@ Summary
 
 # Open Questions
 - If an extension package defines more than one TfM, the driver will just use the first one. Should this be configurable. Can't think of any scenarios for this right now.
+- If an extension is using a different framework than the project, can it still run?
```

Comment:
no, I don't see a reason. But I don't see a way for an extension to define the TFM it runs under for this project